include/UnitTestGroup.h

```diff
@@ -288,6 +288,39 @@ namespace Mezzanine
                 }
             }
 
+            /// @copydoc Test
+            /// @brief Test if an expected range of values contains the given value.
+            /// @param ExpectedLowerBound The Minimum of the target range exclusive.
+            /// @param ExpectedUpperBound The Maximum of the target range exclusive.
```

Comment:
The use of <= and >= in the code below suggests the check is inclusive rather than exclusive.

Reply:
Yeah, Same brain damage incompleteness I re-wrote the whole section.